Resources/services/services.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+
+
+    <services>
+        <service class="MollieShopware\Services\MollieOrderRequestAnonymizer\MollieOrderRequestAnonymizer" id="mollie_shopware.services.mollie_order_request_anonymizer.mollie_order_request_anonymizer" public="true">
+            <argument type="string">***</argument>
+        </service>
+
+        <service class="MollieShopware\Services\Mollie\Payments\Extractor\PaymentFailedDetailExtractor" id="mollie_shopware.services.mollie.payments.extractor.payment_failed_detail_extractor" public="true"/>
```

Comment:
maybe think about this one too, but doesnt look that redundant in the wordins as above

still, does it have to be public?